### ISSUE TITLE
unable to download JSON metadata

### ISSUE THREAD

**Issue by imrankhan441** [rg3/youtube-dl#15141]
ERROR: Unable to download JSON metadata: HTTP Error 403: Forbidden (caused by HTTPError()); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpiyv5xz5s\build\youtube_dl\extractor\common.py", line 517, in _request_webpage
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpiyv5xz5s\build\youtube_dl\YoutubeDL.py", line 2198, in urlopen
  File "C:\Python\Python34\lib\urllib\request.py", line 470, in open
  File "C:\Python\Python34\lib\urllib\request.py", line 580, in http_response
  File "C:\Python\Python34\lib\urllib\request.py", line 508, in error
  File "C:\Python\Python34\lib\urllib\request.py", line 442, in _call_chain
  File "C:\Python\Python34\lib\urllib\request.py", line 588, in http_error_default

**Comment by yan12125** [comment]
Please paste full log, not the last few lines only.

**Comment by imrankhan441** [comment]
[debug] Encodings: locale cp1252, fs mbcs, out cp437, pref cp1252
[debug] youtube-dl version 2017.12.31
[debug] Python version 3.4.4 - Windows-10-10.0.16299
[debug] exe versions: none
[debug] Proxy map: {}
[pluralsight:course] advanced-python: Downloading course JSON
[pluralsight:course] advanced-python: Downloading JSON metadata
ERROR: Unable to download JSON metadata: HTTP Error 403: Forbidden (caused by HTTPError()); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpiyv5xz5s\build\youtube_dl\extractor\common.py", line 517, in _request_webpage
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpiyv5xz5s\build\youtube_dl\YoutubeDL.py", line 2198, in urlopen
  File "C:\Python\Python34\lib\urllib\request.py", line 470, in open
  File "C:\Python\Python34\lib\urllib\request.py", line 580, in http_response
  File "C:\Python\Python34\lib\urllib\request.py", line 508, in error
  File "C:\Python\Python34\lib\urllib\request.py", line 442, in _call_chain
  File "C:\Python\Python34\lib\urllib\request.py", line 588, in http_error_default

**Comment by yan12125** [comment]
The first four lines should be included, like this:
```
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', 'https://www.youtube.com/watch?v=TcXFY-_rDzQ']
```

**Comment by trungdn0401** [comment]
Same problem on this day.
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--verbose', '--user-agent', 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:57.0) Gecko/20100101 Firefox/57.0', '--cookies', '.\\cookies.txt', '--username', 'PRIVATE', '--password', 'PRIVATE', '-f', 'best', '--all-subs', 'https://app.pluralsight.com/library/courses/modernizing-dotnet-framework-apps-docker']
[debug] Encodings: locale cp1252, fs mbcs, out cp437, pref cp1252
[debug] youtube-dl version 2017.12.31
[debug] Python version 3.4.4 - Windows-10-10.0.16299
[debug] exe versions: none
[debug] Proxy map: {}
[pluralsight:course] modernizing-dotnet-framework-apps-docker: Downloading course JSON
[pluralsight:course] modernizing-dotnet-framework-apps-docker: Downloading JSON metadata
ERROR: Unable to download JSON metadata: HTTP Error 403: Forbidden (caused by HTTPError()); please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpiyv5xz5s\build\youtube_dl\extractor\common.py", line 517, in _request_webpage
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpiyv5xz5s\build\youtube_dl\YoutubeDL.py", line 2198, in urlopen
  File "C:\Python\Python34\lib\urllib\request.py", line 470, in open
    elif condition == "open":
  File "C:\Python\Python34\lib\urllib\request.py", line 580, in http_response
    for HTTP, FTP and when applicable HTTPS.
  File "C:\Python\Python34\lib\urllib\request.py", line 508, in error
    def open(self, fullurl, data=None, timeout=socket._GLOBAL_DEFAULT_TIMEOUT):
  File "C:\Python\Python34\lib\urllib\request.py", line 442, in _call_chain
    self.handle_error = {}
  File "C:\Python\Python34\lib\urllib\request.py", line 588, in http_error_default
    FTPHandler, FileHandler, HTTPErrorProcessor,

**Comment by yan12125** [comment]
Thanks for the log. Looks like a duplicate of #14098.

**Comment by dstftw** [comment]
Most likely your IP is blocked by pluralsight.
